bot/exts/info/information.py

```diff
@@ -547,7 +547,21 @@ async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
         for rule in rules:
             self.bot.stats.incr(f"rule_uses.{rule}")
 
-        final_rules = tuple(f"**{pick}.** {full_rules[pick - 1]}" for pick in rules)
+        if rules:
+            final_rules = tuple(f"**{pick}.** {full_rules[pick - 1][0]}" for pick in rules)
+        else:
+            final_rules = tuple(f"**{pick + 1}** {full_rules[pick][0]}" for pick, rule in enumerate(full_rules)
+                                if keyword in rule[1])
+
+        if not rules and not final_rules:
+            # This would mean that we didn't find a match for the used keyword
+            rules_embed.description = (
+                f"There are currently no rules that correspond to keyword: {keyword}."
+                " If you think it should be added, please ask our admins and they'll take care of the rest."
+            )
+
+            await ctx.send(embed=rules_embed)
```

Comment:
For consistency, this error should not be sent as an embed because the invalid indices message doesn't use one:

https://github.com/python-discord/bot/blob/ddd829fc235b0484064fce0c4924e980659ec2bb/bot/exts/info/information.py#L543-L545